groups/sig-node/groups.yaml

```diff
@@ -19,6 +19,19 @@ groups:
   # subproject artifacts being stored in a given staging project
   #
 
+  - email-id: sig-node-leads@kubernetes.io
+    name: sig-node-leads
+    description: |-
+      SIG Node chairs and technical leads comms
+    settings:
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      ReconcileMembers: "true"
+    owners:
+      - dawnchen@google.com
+      - decarr@redhat.com
+      - mpatel@redhat.com
+      - S.Kanzhelev@gmal.com
```

Comment:
This looks like a typo -- this should likely be all lower case, and `gmail.com`

Reply:
thank you for catching!